isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -1764,47 +1764,14 @@ namespace isobus
 		                                                      CANIdentifier::PriorityLowest7);
 	}
 
-	bool VirtualTerminalClient::send_working_set_maintenance(bool initializing, VTVersion workingSetVersion) const
+	bool VirtualTerminalClient::send_working_set_maintenance(bool initializing) const
 	{
 		std::uint8_t versionByte;
 		std::uint8_t bitmask = (initializing ? 0x01 : 0x00);
 
-		switch (workingSetVersion)
-		{
-			case VTVersion::Version3:
-			{
-				versionByte = 0x03;
-			}
-			break;
-
-			case VTVersion::Version4:
-			{
-				versionByte = 0x04;
-			}
-			break;
-
-			case VTVersion::Version5:
-			{
-				versionByte = 0x05;
-			}
-			break;
-
-			case VTVersion::Version6:
-			{
-				versionByte = 0x06;
-			}
-			break;
-
-			default:
-			{
-				versionByte = 0xFF;
-			}
-			break;
-		}
-
 		const std::array<std::uint8_t, CAN_DATA_LENGTH> buffer = { static_cast<std::uint8_t>(Function::WorkingSetMaintenanceMessage),
 			                                                         bitmask,
-			                                                         versionByte,
+			                                                         0x06,
```

Comment:
Nitpick(non-blocking): instead of specifying it directly inside the function, try to define it using a constexpr before so it is clear what it is for later on. 

E.g. "static constexpr std::uint8_t SUPPORTED_VT_VERSION = 0x06"

Reply:
Will do! Thanks.

Reply:
Almost there, only instead of putting it inside the global namespace, I recommend putting it inside the function itself, since that would be the only place this constant will be used. We try to minimize the global namespace usage

Reply:
Alright, will do that and submit a commit shortly.